### PR TITLE
Allow multiple notifications to be sent

### PR DESCRIPTION
**Pull request by MrWoWander**
Adds a new `batchSend` API to allow multiple notifications to be sent with the same call

### CODE REVIEW

**Comment by 0xTim** [comment]
@kylebrowning any thoughts on this? I'm easy either way, it might be useful but not a huge amount of logic for users to implement

**Comment by kylebrowning** [comment]
This works!

File: Sources/APNS/Application+APNS.swift [hunk 1]
```diff
@@ -60,6 +59,52 @@ extension Application.APNS: APNSwiftClient {
         self.application.eventLoopGroup.next()
     }
 
+    public func batchSend(
+        rawBytes payload: ByteBuffer,
+        pushType: APNSwiftConnection.PushType,
+        to deviceToken: String...,
```
Comment (kylebrowning) [inline Sources/APNS/Application+APNS.swift:65]: Any reason we use `to deviceToken: String...,` here and then `to deviceToken: [String],` below?

They are semantically the same yeah? Might as well use the same thing in both cases.

**Review by kylebrowning — DISMISSED** [review]

File: Sources/APNS/Application+APNS.swift (hunk 1 above)
Comment (MrWoWander) [reply Sources/APNS/Application+APNS.swift:65]: Yes. It's just that in some situations it might be more convenient to enumerate, so I added it too

Comment (0xTim) [reply Sources/APNS/Application+APNS.swift:65]: I agree we should probably offer both. Swift doesn't like converting between the two

**Review by 0xTim — APPROVED** [review]

**Comment by philipengberg** [comment]
Am I misunderstanding the implementation of this? It seems that, in reality, there's no difference in just using the regular `send` in a `for` loop compared to this. At the end of it, `batchSend` also just sends **one** push notification at a time. I need to send 250,000 at the *same time* in parallel, not sequentially. How can this be achieved with this library?

**Comment by kylebrowning** [comment]
You can't do that with APNS. You have an open connection with Apple so send as many as you want in a for loop with tasks. 

I've sent 1 million in 10 seconds.

**Pull request merged**